contracts/src/Teleporter/ITeleporterMessenger.sol

```diff
@@ -225,16 +208,13 @@ interface ITeleporterMessenger {
      * @dev Gets the fee token address and amount for a given message.
      * @return The fee token address and fee amount for a the given message.
      */
-    function getFeeInfo(
-        bytes32 destinationBlockchainID,
-        uint256 messageID
-    ) external view returns (address, uint256);
+    function getFeeInfo(bytes32 messageID) external view returns (address, uint256);
```

Comment:
can you add a comment about what happens if a messageID isn't found.

Reply:
Added